lib/src/tileset/tile.dart

```diff
@@ -52,7 +52,7 @@ class Tile {
       terrain: parser
               .getStringOrNull('terrain')
               ?.split(',')
-              .map(int.parse)
+              .map(int.tryParse)
               .toList() ??
```

Comment:
```suggestion
              ?.map(int.tryParse)
              ?.toList() ??
```
It's due to the `terrain` array being null right? Not that it contains unparsable data.

Reply:
Well, I had a tileset from an artist, and the terrain looked like this:
<tile id="0" terrain=",,,0"/>
but now that I've opened it and saved it with the latest version of tiled, that terrain array is no longer there, and looks like is replaced by wangsets, so the bug is no longer applicable. However, this did let me load the tilesets before I had updated them with tiled.